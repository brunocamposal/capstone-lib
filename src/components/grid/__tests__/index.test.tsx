import React from 'react'
import Grid from '../index'
import Col from '../../col/index'
import renderer from 'react-test-renderer'

it('renders correctly grid', () => {
  const tree = renderer
    .create(
      <Grid>
        <Col width={12} backgroundColor='black'>
        </Col>
      </Grid>
    )
    .toJSON()

  expect(tree).toMatchSnapshot()
})
