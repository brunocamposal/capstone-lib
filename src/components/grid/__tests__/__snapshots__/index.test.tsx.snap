// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly grid 1`] = `
<div
  className="sc-bdnylx ciuggH"
>
   
  <div
    className="sc-gtssRu iIsXHW"
    width={12}
  />
   
</div>
`;
